src/api/store/community.py

```diff
@@ -246,26 +244,17 @@ def _update_locations(self, geography_type, locations, community):
                     print("Number of zipcodes = " + str(len(zips)))
                     if len(zips) > 0:
                         for zip in zips:
-                            print(zip)
                             zipcode = zip["zip_code"]
```

Comment:
I suggest you use `zip.get("zip_code")`.  `zip["zip_code"]` will throw errors if, for some reason, `zip_code` is not in the `zip` dictionary

Reply:
done